lib/flutter_config.dart

```diff
@@ -17,26 +17,24 @@ class FlutterConfig {
 
   /// Variables need to be loaded on app startup, recommend to do it `main.dart`
   static loadEnvVariables() async {
-    final Map<String, dynamic> variables =
+    final Map<String, dynamic>? variables =
```

Comment:
Given the `late` operator mentioned above, the Map itself will not be nullable, but the returned variable will be nullable. so this might have to be `Map<String, dynamic?>`

Reply:
Thanks for your comment! Maybe I am missing something, because this call to the platform method channel:
`await _channel.invokeMapMethod('loadEnvVariables');`
returns a `Future<Map<K, V>?>` so the locally declared "variables" needs to be nullable as well.
There's something I am not seeing?

Reply:
Ah, yes sorry I missed that. 

So the returned Map is nullable. `final Map<String, dynamic>? variables = await _channel.invokeMapMethod('loadEnvVariables');` 

but the locally declared "variables" is not `_instance._variables = variables ?? {};`

so I think this should work fine

Might be a good idea to change `variables` to `loadedVariables` for clarity so assigning it to the instance variables will be `_instance._variables = loadedVariables ?? {};`

Reply:
Yes that indeed improves readability. PR Updated!